java/src/main/java/com/codurance/training/tasks/Uncheck.java

```diff
@@ -0,0 +1,19 @@
+package com.codurance.training.tasks;
+
+import java.io.PrintWriter;
+import java.util.List;
+import java.util.Map;
+
+public class Uncheck implements CheckOrUncheckService{
+
+
+
+    @Override
+    public void check(Map<String, List<Task>> tasks, String idString, PrintWriter out) {
+    }
```

Comment:
according to Interface Segregation principle we should not force a class to implement a method which is not functional
Either merge Check and Uncheck classes 
or segregate interface